vllm/config.py

```diff
@@ -71,6 +71,8 @@ class ModelConfig:
         max_context_len_to_capture: Maximum context len covered by CUDA graphs.
             When a sequence has context length larger than this, we fall back
             to eager mode.
+        disable_tokenizer: If true, tokenization and detokenization are
+            disabled.
```

Comment:
nit: IMO this is slightly misleading because we're not simply disabling tokenization and detokenization but in fact the tokenizers and detokenizers are not initialized at all.

Reply:
done